src/ui/units/dash/containers/Dialogs/GroupControl/ControlsPlacementDialog/ControlsPlacementDialog.tsx

```diff
@@ -71,7 +71,10 @@ const ControlsPlacementDialog = ({onClose}: ControlsPlacementDialogProps) => {
             const dragItem = prevItemsState[oldIndex];
             const newItemsState = prevItemsState.filter((_, index) => index !== oldIndex);
             newItemsState.splice(newIndex, 0, dragItem);
-            return newItemsState;
+            return {
+                ...prevItemsState,
+                items: newItemsState,
```

Comment:
Is there a `items` field definitely present in `itemsState`? The initial value is `selectorsGroup.items`

Reply:
Oh, my bad, thank you